### PR TITLE
Doc storybook stories

### PR DESCRIPTION
**Pull request by Yopo-Nicolas**

### CODE REVIEW

File: src/Components/SFChipsListField/SFChipsListField.stories.tsx [hunk 1]
```diff
@@ -112,10 +112,60 @@ const ChipListOptions = [
...
+        'If true, the chiplistfield is free solo, meaning that the user input is not bound to provided options.'
+    },
+    helperText: {
+      description: 'Text to help understand input values.'
+    },
+    inputMinWidth: {
+      description: 'Minimmum with used by the input.'
+    },
+    inputType: {
+      description: 'Type of value the input can receive.'
```
Comment (pdcastano) [inline src/Components/SFChipsListField/SFChipsListField.stories.tsx:141]: The type of the inner input component field

**Review by pdcastano — CHANGES_REQUESTED** [review]

File: src/Components/SFChip/SFChip.stories.tsx [hunk 2]
```diff
@@ -15,6 +16,23 @@ export default {
...
+    deleteable: {
+      description:
+        'If true, the chip shows the delete icon and the chip could be deleted.'
+    },
+    fullWidth: {
+      description:
+        'If true, the component will take up the full width of its container.'
+    },
+    hasError: {
+      description: '	If true, the chip should be displayed in a error state.'
```
[the remark this answers is not in the document]
Comment (Yopo-Nicolas) [reply src/Components/SFChip/SFChip.stories.tsx]: done

File: src/Components/SFChipsListField/SFChipsListField.stories.tsx [hunk 3]
```diff
@@ -112,10 +112,60 @@ const ChipListOptions = [
...
+    },
+    disabled: {
+      description: 'If true, the chip should be displayed in a disabled state.'
+    },
+    emptyMessage: {
+      description: 'Message displayed when the input is empty.'
+    },
+    freeSolo: {
+      description:
+        'If true, the chiplistfield is free solo, meaning that the user input is not bound to provided options.'
```
[the remark this answers is not in the document]
Comment (Yopo-Nicolas) [reply src/Components/SFChipsListField/SFChipsListField.stories.tsx]: done

File: src/Components/SFChipsListField/SFChipsListField.stories.tsx [hunk 4]
```diff
@@ -112,10 +112,60 @@ const ChipListOptions = [
...
+    },
+    helperText: {
+      description: 'Text to help understand input values.'
+    },
+    inputMinWidth: {
+      description: 'Minimmum with used by the input.'
```
[the remark this answers is not in the document]
Comment (Yopo-Nicolas) [reply src/Components/SFChipsListField/SFChipsListField.stories.tsx]: done

File: src/Components/SFChipsListField/SFChipsListField.stories.tsx (hunk 1 above)
Comment (Yopo-Nicolas) [reply src/Components/SFChipsListField/SFChipsListField.stories.tsx]: done

File: src/Components/SFChipsListField/SFChipsListField.stories.tsx [hunk 5]
```diff
@@ -112,10 +112,60 @@ const ChipListOptions = [
...
+    },
+    inputType: {
+      description: 'Type of value the input can receive.'
+    },
+    isEditable: {
+      description: 'If true, the content of the chip is able to be edited.'
+    },
+    itemChipDisplay: {
+      description: 'How the saved chip items should be displayed.'
```
[the remark this answers is not in the document]
Comment (Yopo-Nicolas) [reply src/Components/SFChipsListField/SFChipsListField.stories.tsx]: done

File: src/Components/SFChipsListField/SFChipsListField.stories.tsx [hunk 6]
```diff
@@ -112,10 +112,60 @@ const ChipListOptions = [
...
+    },
+    items: {
+      description: 'List of chip items saved.',
+      control: false
+    },
+    label: {
+      description: 'The content of the label.'
+    },
+    options: {
+      description: 'Array of options.',
```
[the remark this answers is not in the document]
Comment (Yopo-Nicolas) [reply src/Components/SFChipsListField/SFChipsListField.stories.tsx]: done

**Review by pdcastano — APPROVED** [review]

**Comment by pdcastano** [comment]
LGTM

**Pull request merged**